data/seeds/001projects.js

```diff
@@ -0,0 +1,12 @@
+
+exports.seed = function (knex, Promise) {
+  return knex('projects')
+    .truncate()
+    .then(function () {
+      return knex('projects').insert([
+        { name: 'Project 1', description: 'Project 1 description' },
+        { name: 'Project 2', description: 'Project 2 description' },
+        { name: 'Project 3', description: 'project 3 description' }
```

Comment:
Don't you think we could reduce the repetition in the creation of this array? For example:
```javascript
let i = 1;
return knex('projects').insert(new Array(3)
  .fill(null)
  .map((e, i) => ({ 
    name: `Project ${++i}`, 
    description: `Project ${i} description` 
  }))
);
```